spec/controllers/vm_controller_spec.rb

```diff
@@ -20,4 +20,26 @@
       expect(response).to render_template('vm_common/_live_migrate')
     end
   end
+
+  describe "#right_size" do
+    before do
+      stub_user(:features => :all)
+      @vm = FactoryBot.create(:vm_vmware)
```

Comment:
Instance variables in rspec can tamper with other specs, please use this outside the `before` block.
```suggestion
      let!(:vm) { FactoryBot.create(:vm_vmware) }
```